.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
[note] This is used to disable the ability to create a blank issue, and force a template to be used.

We also added the generic 'other' template that is basically blank, except still includes the warning about creating issues on the public repo.